subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -798,43 +798,38 @@ static int cmd_wifi_status(const struct shell *sh, size_t argc, char *argv[])
 	context.sh = sh;
 
 	if (net_mgmt(NET_REQUEST_WIFI_IFACE_STATUS, iface, &status,
-				sizeof(struct wifi_iface_status))) {
-		shell_fprintf(sh, SHELL_WARNING, "Status request failed\n");
+		     sizeof(struct wifi_iface_status))) {
+		PR_WARNING("Status request failed\n");
 
 		return -ENOEXEC;
 	}
 
-	shell_fprintf(sh, SHELL_NORMAL, "Status: successful\n");
-	shell_fprintf(sh, SHELL_NORMAL, "==================\n");
-	shell_fprintf(sh, SHELL_NORMAL, "State: %s\n", wifi_state_txt(status.state));
+	PR("Status: successful\n");
+	PR("==================\n");
+	PR("State: %s\n", wifi_state_txt(status.state));
 
 	if (status.state >= WIFI_STATE_ASSOCIATED) {
 		uint8_t mac_string_buf[sizeof("xx:xx:xx:xx:xx:xx")];
 
-		shell_fprintf(sh, SHELL_NORMAL, "Interface Mode: %s\n",
-				wifi_mode_txt(status.iface_mode));
-		shell_fprintf(sh, SHELL_NORMAL, "Link Mode: %s\n",
-				wifi_link_mode_txt(status.link_mode));
-		shell_fprintf(sh, SHELL_NORMAL, "SSID: %.32s\n", status.ssid);
-		shell_fprintf(sh, SHELL_NORMAL, "BSSID: %s\n",
-					  net_sprint_ll_addr_buf(status.bssid,
-					  WIFI_MAC_ADDR_LEN, mac_string_buf,
-					  sizeof(mac_string_buf))
-					 );
-		shell_fprintf(sh, SHELL_NORMAL, "Band: %s\n",
-				wifi_band_txt(status.band));
-		shell_fprintf(sh, SHELL_NORMAL, "Channel: %d\n", status.channel);
-		shell_fprintf(sh, SHELL_NORMAL, "Security: %s\n",
-				wifi_security_txt(status.security));
-		shell_fprintf(sh, SHELL_NORMAL, "MFP: %s\n",
-				wifi_mfp_txt(status.mfp));
+		PR("Interface Mode: %s\n", wifi_mode_txt(status.iface_mode));
+		PR("Link Mode: %s\n", wifi_link_mode_txt(status.link_mode));
+		PR("SSID: %.32s\n", status.ssid);
+		PR("BSSID: %s\n",
+		   net_sprint_ll_addr_buf(status.bssid,
+		   WIFI_MAC_ADDR_LEN, mac_string_buf,
+		   sizeof(mac_string_buf))
+		  );
```

Comment:
Since we're already touching those lines I suggest to modify the alignment a bit, so that it's clear where each argument belongs. 
```suggestion
		PR("BSSID: %s\n",
		   net_sprint_ll_addr_buf(status.bssid, 
		   			  WIFI_MAC_ADDR_LEN, mac_string_buf,
		   			  sizeof(mac_string_buf)));
```

Reply:
The changes have been incorporated and squashed into this commit: https://github.com/zephyrproject-rtos/zephyr/pull/68185/commits/236963e6c6f4245fadb2b34c7729f3bf3dbdc230